Source/WebCore/Modules/speech/SpeechRecognitionResultList.cpp

```diff
@@ -0,0 +1,33 @@
+
+#include "config.h"
+
+#if ENABLE(SPEECH_RECOGNITION)
+
+#include "SpeechRecognitionResultList.h"
+
+namespace WebCore {
+PassRefPtr<SpeechRecognitionResultList> SpeechRecognitionResultList::create(const Vector<RefPtr<SpeechRecognitionResult> >& results)
+{
+    LOG(SpeechRecognition, "%s:%s:%d \n\n", __FILE__, __func__, __LINE__);
+
+    return adoptRef(new SpeechRecognitionResultList(results));
```

Comment:
Methods like these, which don't return a null pointer in any outcome, should have Ref<> as a return value. If a null value is possible, the return value should be RefPtr<>. PassRefPtr<> is not really used anymore and is being phased out.